src/mod/event_handlers/mod_json_cdr/mod_json_cdr.c

```diff
@@ -319,8 +320,10 @@ static void process_cdr(cdr_data_t *data)
 								switch_core_new_memory_pool(&pool);
 								if(pool && switch_file_copy(path, move_path, SWITCH_FPROT_FILE_SOURCE_PERMS, pool) == SWITCH_STATUS_SUCCESS) {
 									switch_log_printf(SWITCH_CHANNEL_UUID_LOG(data->uuid), SWITCH_LOG_INFO, "Move CDR [%s] to [%s]\n", data->filename, data->logdir);
+									switch_file_remove(path, pool);
```

Comment:
Not sure on this line, why adding the switch_file_remove?

Reply:
This is copy-delete fallback if in case move failed (moving file to a different mount or filesystem will cause `Invalid cross-device link` error). So once we are able to copy the tmp cdr to the cdr folder we need to clean up the files.